worlds/Files.py

```diff
@@ -33,9 +32,9 @@ def __new__(mcs, name: str, bases: Tuple[type, ...], dct: Dict[str, Any]) -> Aut
 
     @staticmethod
     def get_handler(file: str) -> Optional[AutoPatchRegister]:
-        for file_ending, handler in AutoPatchRegister.file_endings.items():
-            if file.endswith(file_ending):
-                return handler
+        _, suffix = os.path.splitext(file)
+        if suffix in AutoPatchRegister.file_endings:
+            return AutoPatchRegister.file_endings[suffix]
         return None
```

Comment:
sorry I should've noticed this earlier. I think it's a tiny bit slower but it reads a lot nicer imo
```suggestion
        return AutoPatchRegister.file_endings.get(suffix, None)
```